simulation.py

```diff
@@ -0,0 +1,79 @@
+"""
+Fuctions for a process to simulate an alpha.
+"""
+
+import copy
+import importlib
+import sys
+from thousandaire.constants import DATA_LIST_ALL
+from thousandaire.constants import TRADING_CONFIGS, TRADING_REGIONS
+from thousandaire.data_loader import DataLoader
+from thousandaire.simulator import Simulator
+
+PRICE_DATASET = 'price_dataset'
+
+def initialize():
+    """
+    Initialize data we get.
+    """
+    raw_data = DataLoader(DATA_LIST_ALL).get_all()
+    workdays_all = {
+        region : raw_data['workdays'][region]
+        for region in TRADING_REGIONS if region in raw_data['workdays']}
+    return_data = {}
+    for region, workdays in workdays_all.items():
+        return_data[region] = copy.deepcopy(raw_data)
+        for dataset in return_data[region].values():
+            dataset.set_workdays(workdays)
+    return_data['workdays'] = workdays_all
+    return return_data
+
+def extract_data(raw_data, data_list, price_dataset, region):
+    """
+    Extract the data which alpha need.
+
+    Because the raw_data is combined with all data we have, we need to pure
+    the data ,which the alpha need, to pass for saving resource.
+    """
+    workdays = raw_data['workdays'][region]
+    target_data = raw_data[region][price_dataset]
+    simulate_data = {
+        name : raw_data[region][name]
+        for name in data_list if name in raw_data[region]}
+    data = {
+        'workdays' : workdays,
+        'target_data' : target_data,
+        'simulate_data' : simulate_data}
+    return data
+
+def simulate(alpha_path):
+    """
+    Act the process of simulating.
+    """
+    try:
+        settings = importlib.import_module(alpha_path).AlphaSettings()
+    except FileNotFoundError as error:
+        raise FileNotFoundError("Alpha does not exit.") from error
+    raw_data = initialize()
+    _, region = settings.target
+    data = extract_data(
+        raw_data, settings.data_list,
+        TRADING_CONFIGS[settings.target][PRICE_DATASET], region)
+    simulator = Simulator(settings, data)
+    results = simulator.run()
+    return results
+
+def main():
+    """
+    Run the simulation process.
+    """
+    args = sys.argv
+    target = 'settings'
+    for item in args:
+        if item.find(target) != -1:
+            alpha_path = item
```

Comment:
Use [Argparse](https://docs.python.org/3/howto/argparse.html) to deal with system arguments.